src/Microsoft.Build.Sql/sdk/Sdk.targets

```diff
@@ -68,6 +68,28 @@
   <Target Name="_CheckForUnsupportedTargetFrameworkAndFeatureCombination" />
 
   <UsingTask Condition="'$(NetCoreBuild)' == 'true'" TaskName="AllowEmptyTelemetry" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <UsingTask Condition="'$(NetCoreBuild)' == 'true'" TaskName="CheckForDuplicateItems" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <Target Name="CheckForDuplicateSqlItems" BeforeTargets="_SetupSqlBuildInputs">
+
+    <PropertyGroup>
+      <DefaultItemsMoreInformationLink>https://learn.microsoft.com/sql/tools/sql-database-projects/howto/convert-original-sql-project</DefaultItemsMoreInformationLink>
```

Comment:
@dzsquared Do you think this is the appropriate link to show in the error message? For reference the .NET SDK uses this link: https://aka.ms/sdkimplicititems